nix/flake.nix

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Can you add a `BUILD.bazel` file to this package and a README explaining what the package is for?